src/attestation/README.md

```diff
@@ -1,122 +1,74 @@
 
-#### Add a new attestation policy certificate ####
-Example:
+Content:
 ```
-az attestation signer add \
--n "myattestationprovider" -g "MyResourceGroup" \
---signer "eyAiYWxnIjoiUlMyNTYiLCAie..."
+version= 1.0;
+authorizationrules {
+[ type=="$is-debuggable", value==false ]
+&& [ type=="$product-id", value==4639 ]
+&& [ type=="$min-svn", value>= 0 ]
+&& [ type=="$sgx-mrsigner", value=="E31C9E505F37A58DE09335075FC8591254313EB20BB1A27E5443CC450B6E33E5"]
+=> permit();
+};
+ issuancerules {
+c:[ type=="$sgx-mrsigner" ] => issue(type="sgx-mrsigner",
+value=c.value);
+ c1:[type=="maa-ehd"] => issue(type="aas-ehd", value=c1.value);
+ };
 ```
-
-#### Remove the specified policy management certificate ####
-Example:
+Run the command:
 ```
-az attestation signer remove \
--n "myattestationprovider" -g "MyResourceGroup" \
---signer "eyAiYWxnIjoiUlMyNTYiLCAie..."
+az attestation policy set -n testatt1 -g testrg --attestation-type SGX-IntelSDK -f "{local_path}\text_sgx_policy.txt"
 ```
-
-#### Retrieve the set of certificates used to express policy ####
-Example:
+#### Get policy ####
 ```
-az attestation signer list \
--n "myattestationprovider" -g "MyResourceGroup"
+az attestation policy show -n testatt1 -g testrg --attestation-type SGX-IntelSDK
 ```
+#### Configure policy in unsigned JWT format using file path ####
+Download the policy file: https://github.com/Azure/azure-cli-extensions/blob/master/src/attestation/azext_attestation/tests/latest/policies/unsigned_jwt_sgx_policy.txt
 
-#### Set the policy for a given kind of TEE ####
-Note: You need to specify `-n` and `-g` (or use `-u`) even if they are not marked as `required` parameters.
-
-Example:
+Content:
 ```
-az attestation policy set \
--n "myattestationprovider" -g "MyResourceGroup" \
---tee SgxEnclave --new-attestation-policy "newAttestationPolicyname"
-
-az attestation policy set \
--u https://myattestationprovider.eastus2.attest.azure.net \
---tee SgxEnclave --new-attestation-policy "newAttestationPolicyname"
+eyJhbGciOiJub25lIn0.eyJBdHRlc3RhdGlvblBvbGljeSI6ICJkbVZ5YzJsdmJqMGdNUzR3TzJGMWRHaHZjbWw2WVhScGIyNXlkV3hsYzN0ak9sdDBlWEJsUFQwaUpHbHpMV1JsWW5WbloyRmliR1VpWFNBOVBpQndaWEp0YVhRb0tUdDlPMmx6YzNWaGJtTmxjblZzWlhON1l6cGJkSGx3WlQwOUlpUnBjeTFrWldKMVoyZGhZbXhsSWwwZ1BUNGdhWE56ZFdVb2RIbHdaVDBpYVhNdFpHVmlkV2RuWVdKc1pTSXNJSFpoYkhWbFBXTXVkbUZzZFdVcE8yTTZXM1I1Y0dVOVBTSWtjMmQ0TFcxeWMybG5ibVZ5SWwwZ1BUNGdhWE56ZFdVb2RIbHdaVDBpYzJkNExXMXljMmxuYm1WeUlpd2dkbUZzZFdVOVl5NTJZV3gxWlNrN1l6cGJkSGx3WlQwOUlpUnpaM2d0YlhKbGJtTnNZWFpsSWwwZ1BUNGdhWE56ZFdVb2RIbHdaVDBpYzJkNExXMXlaVzVqYkdGMlpTSXNJSFpoYkhWbFBXTXVkbUZzZFdVcE8yTTZXM1I1Y0dVOVBTSWtjSEp2WkhWamRDMXBaQ0pkSUQwLUlHbHpjM1ZsS0hSNWNHVTlJbkJ5YjJSMVkzUXRhV1FpTENCMllXeDFaVDFqTG5aaGJIVmxLVHRqT2x0MGVYQmxQVDBpSkhOMmJpSmRJRDAtSUdsemMzVmxLSFI1Y0dVOUluTjJiaUlzSUhaaGJIVmxQV011ZG1Gc2RXVXBPMk02VzNSNWNHVTlQU0lrZEdWbElsMGdQVDRnYVhOemRXVW9kSGx3WlQwaWRHVmxJaXdnZG1Gc2RXVTlZeTUyWVd4MVpTazdmVHMifQ.
```

Comment:
this is fake jwt plicy right

Reply:
@yungezz It's a valid policy structure but without any sensitive data.